index.html

```diff
@@ -1,12 +1,145 @@
 <!DOCTYPE html>
 <html lang="ru">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <a class="logo" href="index.html"
+          ><img class="logo" src="#" alt="Logo"
+        /></a>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="index.html">Главная</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">О седоне</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="catalog.html">Гостиницы</a>
+          </li>
+        </ul>
+        <ul class="navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#"><img src="#" alt="Поиск" /></a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#"
+              ><img src="#" alt="Избранное"
+            /></a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Хочу сюда!</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-index">
+      <h1 class="visually-hidden">Sedona</h1>
```

Comment:
Тут лучше сделать секцию с название с классом hero и положить туда h1 и картинку)

```html
 <section class="hero">
     <h1 class="visually-hidden">Главная страница о туризме в городе Седона</h1>
     <img src="./images/welcome.svg" alt="Welocome to the gorgeous Sedona Because thr Grand Canyon sucks!">
</section>

```